src/utilities/icon.scss

```diff
@@ -1,7 +1,84 @@
 @use './defaults.scss' as *;
 
-// Import Font Awesome Icons
-@import url('https://unpkg.com/font-awesome/css/font-awesome.min.css');
+// GC Design System Fonts - Icons
+@font-face {
+  font-family: "gcds-icons";
+  src: url("../../node_modules/@cdssnc/gcds-fonts/fonts/icons/gcds-icons.ttf") format("truetype"),
```

Comment:
I am wondering if we may have to do something similar to what we did in the components repo and include an assets folder with the font files since if someone uses the UF by CDN there won't be a `node_modules` directory available.